templates/definition/meter/huawei-sun2000.yaml

```diff
@@ -8,13 +8,37 @@ capabilities: ["battery-control"]
 requirements:
   description:
     de: |
-      Grid und Batterie erfordern den PowerSensor.
-      "Modbus/TCP" erfordert Freischaltung via "Errichterzugang" in den Kommunikationseinstellungen des Wechselrichters. 
+      Netz und Batterie erfordern den PowerSensor.
+
+      Die Verbindung zum Wechselrichter erfolgt über Modbus. Die folgenden Details sollten Ihnen bei der Einrichtung der Verbindung helfen.
+
+      - Die Modbus-ID sollte 0 sein, es sei denn, Sie verwenden eine RS485-Verbindung.
```

Comment:
Was bedeutet "sollte"- ist das notwendig? Funktioniert so aktuell ohnehin nicht da 1 bereits als Default gesetzt ist.

Reply:
Address 1 works for me for Modbus TCP via SDongleA and RS485 directly to the SUN2000. Is 0 necessary for the SUN2000's ad-hoc Wi-Fi?